arapp.json

```diff
@@ -22,11 +22,23 @@
       "wsRPC": "wss://rinkeby.eth.aragon.network/ws",
       "network": "rinkeby"
     },
+    "staging": {
+      "registry": "0x98df287b6c145399aaa709692c8d308357bc085d",
```

Comment:
Double checking this address, I get a different one. I looked here: https://github.com/aragon/deployments/blob/master/environments/rinkeby/readme.md

Reply:
I choose the same as Rinkeby cause this: https://github.com/aragon/deployments/issues/54